lib/torba/package.rb

```diff
@@ -157,13 +157,20 @@ def build_import_list
     def process_stylesheets
       import_list.css_assets.each do |asset|
         content = File.read(asset.absolute_path)
+        processed = false
 
         new_content = CssUrlToErbAssetPath.call(content, asset.absolute_path) do |image_file_path|
+          processed = true
           image_asset = import_list.find_by_absolute_path(image_file_path)
           image_asset.logical_path
         end
 
-        new_absolute_path = File.join(load_path, asset.logical_path + ".erb")
+        if processed
```

Comment:
In my local benchmarks comparison is fast even with large strings. The two objects are allocated anyway. So I would go with comparison.